src/os/bluestore/BitAllocator.cc

```diff
@@ -1669,3 +1707,11 @@ void BitAllocator::free_blocks_dis(int64_t num_blocks, ExtentList *block_list)
   alloc_assert(get_used_blocks() >= 0);
   unlock();
 }
+
+void BitAllocator::dump()
+{
+  int count = 0;
+  serial_lock(); 
+  dump_int(count);
```

Comment:
I don't see count is been incremented and passed on, in a loop. BMapAreaIN::dump_int() increments at the end of it.
I might be missing something here. From the current code, i don't see the use of count except passing to child to dump its state. But we are not sending the incremented value.